src/main/java/frc/robot/subsystems/shooter/ShooterIOReal.java

```diff
@@ -0,0 +1,88 @@
+package frc.robot.subsystems.shooter;
+
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.configs.TalonFXConfigurator;
+import com.ctre.phoenix6.controls.MotionMagicVelocityTorqueCurrentFOC;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import edu.wpi.first.units.Angle;
+import edu.wpi.first.units.MutableMeasure;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.units.Velocity;
+
+public class ShooterIOReal implements ShooterIO {
+    private final TalonFX topMotor;
+    private final TalonFX bottomMotor;
+    private final TalonFXConfigurator topMotorConfigurator;
+    private final TalonFXConfigurator bottomMotorConfigurator;
```

Comment:
1. These don't need to be class members. However, I would not create these at all, because they're used only once.

Reply:
fef78d7